### PR TITLE
Feature/neo4j relations

### PR DESCRIPTION
**Pull request by Joostrothweiler**

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -6,6 +6,7 @@ services:
       - "2224:5432"
     volumes:
       - ./data/pg:/var/lib/postgresql/data
+      - ./pgsql_backup:/opt/pgsql_backup
```
Comment (steffansluis) [inline docker-compose.yaml]: I don't this this has to be comitted (nor added to the .gitignore)

File: src/migrations/neo/movies.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    form: model,
+    json: true
+  })
+  console.log('inserted model')
+}
+
+async function insertNeoRole(actor, movie, roles) {
+  let urlEnd = roles == null ? '' :  `?roles="${encodeURIComponent(roles)}"`;
+  return rp.post({
+    url: BASE_URL + `/neo/actors/${actor}/movies/${movie}` + urlEnd,
```
Comment (steffansluis) [inline src/migrations/neo/movies.js]: Does the `/neo` in the URL refer to same thing as the db in the `fetchMovies` function?

File: src/pg/index.js [hunk 3]
```diff
@@ -36,8 +39,13 @@ function _mapIncludes(type) {
 }
 
 async function find(type, id) {
-  let results = await findAll(type, { id });
-  return results[0];
+  console.log(`Finding ${type} by id ${id}`);
+
+  const Model = _getModel(type);
+  return Model.findOne({
```
Comment (steffansluis) [inline src/pg/index.js:45]: :tada:

File: docker-compose.yaml (hunk 1 above)
Comment (Joostrothweiler) [reply docker-compose.yaml]: Removed

File: src/migrations/neo/movies.js (hunk 2 above)
Comment (Joostrothweiler) [reply src/migrations/neo/movies.js]: Yes, added the db as a parameter to all functions.

**Review by steffansluis — APPROVED** [review]

**Pull request merged**